.github/workflows/push-treasurenet-mainnet-artifacts-to-s3.yml

```
@@ -0,0 +1,178 @@
+          bash init_seednode.sh
+
+      - name: Update configurations
+        run: |
+          set -euo pipefail
+          cd ${{ env.SCRIPTS_PATH }}
+          chmod +x change_app.toml.sh change_config.toml.sh
+          bash change_app.toml.sh
+          bash change_config.toml.sh
+
+      - name: Create node archive files
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:8:30: Double quote to prevent globbing and word splitting

(shellcheck)